samples/Basic/Basic/Views/Home/NeedsAuth.cshtml

```diff
@@ -13,7 +13,7 @@
     </head>
     <body>
         <p>You must first log into your LMS.</p>
-        <p>By default this sample points to an LMS hosted at lms.valence.desire2learn.com. You can use the user "sampleapiuser" with the password "Tisabiiif" to log in.</p>
+        <p>By default this sample points to an LMS hosted at devcop.brightspacedemo.com. You can use the user "DevCoPStudent" with the password "/10T41.Vf\0w~=O" to log in.</p>

```

Comment:
Intentionally not pulling from web.config - the user could change their config but this sentence is talking about a user/pass explicitly on devcop.brightspacedemo.com